api/test/wfl/integration/workloads_test.clj

```diff
@@ -0,0 +1,31 @@
+(ns wfl.integration.workloads-test
+  (:require [clojure.test :refer [deftest testing is] :as clj-test]
+            [clojure.test :refer :all]
+            [wfl.jdbc :as jdbc]
+            [wfl.module.all :as all]
+            [wfl.module.copyfile :as copyfile]
+            [wfl.tools.endpoints :as endpoints]
+            [wfl.tools.fixtures :as fixtures]
+            [wfl.tools.workloads :as workloads]))
+
+(clj-test/use-fixtures :once fixtures/temporary-postgresql-database)
+
+(defn ^:private make-copyfile-workload-request
+  [src dst]
+  (-> (workloads/copyfile-workload-request src dst)
+      (assoc :creator (:email @endpoints/userinfo))))
+
+(defn ^:private old-create-copyfile-workload! []
+  (let [request (make-copyfile-workload-request "gs://fake/input" "gs://fake/output")]
+    (jdbc/with-db-transaction [tx (fixtures/testing-db-config)]
+      (let [[id table] (all/add-workload-table! tx copyfile/workflow-wdl request)
+            add-id (fn [m id] (assoc (:inputs m) :id id))]
+        (jdbc/insert-multi! tx table (map add-id (:items request) (range)))
+        (jdbc/update! tx :workload {:version "0.3.8"} ["id = ?" id])
+        table))))
+
+(deftest test-loading-old-copyfile-workload-with-project
```

Comment:
I'm not sure this test is exactly what we want - the "old" copyfile stuff is testing backwards compatibility whereas I would have thought that this test namespace is about querying workloads by project/uuid.
I suggest using `wfl.tools.workloads/create-workload!` to add some of the workloads defined in the name namespce ot the db then query them by project, testing:
- no project returns empty 
- query by project returns only those workloads with that project (case sensitive?)

Reply:
FWIW: I'm for case sensitive.

Reply:
Fair enough, I took `make-copyfile-workload-request` only because I thought it was used for creating workloads, will update!

Reply:
case sensitive test added